src/Nemo.jl

```diff
@@ -43,13 +41,8 @@ import Base: Array, abs, abs2, acos, acosh, asin, asinh, atan, atanh, bin, binom
              typed_hcat, vcat, xor, zero, zeros, +, -, *, ==, ^, &, |, <<, >>,
              ~, <=, >=, <, >, //, /, !=
 
-if VERSION >= v"1.5.0-DEV.639"
-  import Base: contains
-end
-
-if VERSION >= v"1.6.0-DEV.292"
-  import Base: sincospi
-end
+import Base: contains
+import Base: sincospi
```

Comment:
```suggestion
import Base: contains, sincospi
```

Reply:
yeah and in fact it could be merged into the `import` just before...